tendrl/monitoring_integration/alert/utils.py

```diff
@@ -0,0 +1,167 @@
+import pkgutil
+
+from etcd import EtcdKeyNotFound
+from subprocess import CalledProcessError
+from subprocess import check_output
+from tendrl.commons.objects.alert import Alert
+from tendrl.commons.utils import etcd_utils
+from tendrl.commons.utils import log_utils as logger
+from tendrl.monitoring_integration.alert import constants
+from tendrl.monitoring_integration.alert.exceptions import AlertNotFound
+from tendrl.monitoring_integration.alert.exceptions import Unauthorized
+from tendrl.monitoring_integration.alert.exceptions import NodeNotFound
+from tendrl.monitoring_integration.grafana import alert
+
+
+def list_modules_in_package_path(package_path, prefix):
```

Comment:
Is this function similar to https://github.com/Tendrl/commons/blob/master/tendrl/commons/__init__.py#L539 ?

Reply:
yes it is similar